index.html

```diff
@@ -3,12 +3,41 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vanilla Todo</title>
+    <title>🔫 To Do List</title>
     <link rel="stylesheet" href="style.css" />
   </head>
 
   <body>
-    <div class="container"></div>
+    <div class="container">
+      <header>
+        <h2 class="clock">00:00:00</h2>
+        <h2 class="title">🔫 To Do List</h2>
+        <form>
+          <input
+            class="toDoInput"
+            type="text"
+            placeholder="할 일을 입력하세요"
+          />
+          <button class="addToDoBtn">+</button>
+        </form>
+      </header>
+      <div class="list">
+        <div class="toDo">
+          <div>
+            <span>🔮 To Do</span>
+            <span class="toDoNum">(0)</span>
+          </div>
+          <ul class="toDoList"></ul>
+        </div>
+        <div class="done">
+          <span>❤️‍🔥 Done</span>
+          <span class="doneNum">(0)</span>
+          <ul class="doneList"></ul>
```

Comment:
HTML 태그의 id/class 이름은 `kebab-case`로 짓는 게 일반적입니다.

```suggestion
            class="todo-input"
            type="text"
            placeholder="할 일을 입력하세요"
          />
          <button class="add-todo-Btn">+</button>
        </form>
      </header>
      <div class="list">
        <div class="toDo">
          <div>
            <span>🔮 To Do</span>
            <span class="todo-num">(0)</span>
          </div>
          <ul class="todo-list"></ul>
        </div>
        <div class="done">
          <span>❤️‍🔥 Done</span>
          <span class="done-num">(0)</span>
          <ul class="done-list"></ul>
```

Reply:
네! 참고해서 수정해보겠습니다 !! 감사합니다!